lib/i18nPaths.js

```diff
@@ -0,0 +1,10 @@
+const i18nPaths = {
+    en: [
+        "/tools/cli"
+    ],
+    de: [
+        "/tools/cli"
+    ]
+};
```

Comment:
What is the use of these `i18nPaths`?

Reply:
It would work as a record for the translated links. Since we are using static site generation, we needed a mechanism to predefine the translated webpages which could be used by the i18n LinkComponent